src/prob/opf.jl

```diff
@@ -10,6 +10,12 @@ function run_mc_opf(data::Union{Dict{String,<:Any},String}, model_type::Type, so
 end
 
 
+"Run multinetwork optimal power flow problem"
+function run_mn_mc_opf(data::Union{Dict{String,<:Any},String}, model_type::Type, solver; kwargs...)
+    return run_mc_model(data, model_type, solver, build_mn_mc_opf; kwargs...)
```

Comment:
To follow the PowerModels conventions, this call requires `run_model(...; multinetwork=true, kwargs...)`.  The user is not expected to set this one level up.